lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -1648,6 +1730,16 @@ def has_image_changed(self, old_image):
             old_image = old_image.split('@')[0]
         return self.image != old_image, old_image
 
+    def has_rollback_config_changed(self, old_rollback_config):
+        defined_options = dict(
```

Comment:
You should abort early with `True` if `old_rollback_config is None and self.rollback_config`.